app/core/src/main/resources/messages_fr_FR.properties

```diff
@@ -368,36 +368,37 @@ settings.update=Mise à jour disponible
 settings.updateAvailable={0} est la version actuellement installée. Une nouvelle version ({1}) est disponible.
 
 # Update modal and notification strings
-update.urgentUpdateAvailable=🚨 Update Available
-update.updateAvailable=Update Available
-update.modalTitle=Update Available
-update.current=Current
-update.latest=Latest
-update.latestStable=Latest Stable
-update.priority=Priority
-update.recommendedAction=Recommended Action
-update.breakingChangesDetected=⚠️ Breaking Changes Detected
-update.breakingChangesMessage=This update contains breaking changes. Please review the migration guides below.
-update.migrationGuides=Migration Guides:
-update.viewGuide=View Guide
-update.loadingDetailedInfo=Loading detailed version information...
-update.close=Close
-update.viewAllReleases=View All Releases
-update.downloadLatest=Download Latest
-update.availableUpdates=Available Updates:
-update.unableToLoadDetails=Unable to load detailed version information.
+update.urgentUpdateAvailable=🚨 Mise à jour disponible
+update.updateAvailable=Mise à jour disponible
+update.modalTitle=Mise à jour disponible
+update.current=Courante
+update.latest=Dernière
+update.latestStable=Dernière version stable
+update.priority=Priorité
+update.recommendedAction=Action recommandée
+update.breakingChangesDetected=⚠️ Changement critique
+update.breakingChangesMessage=Cette mise à jour contient des changements critiques. Veuillez suivre les indications des guides de migration ci-dessous.
+update.migrationGuides=Guides de Migration :
+update.viewGuide=Lire le Guide
+update.loadingDetailedInfo=Chargement des informations de version détaillées...
+update.close=Fermer
+update.viewAllReleases=Voir toutes les versions
+update.downloadLatest=Télécharger la plus récente
+update.availableUpdates=Mises à jour disponibles:
+update.unableToLoadDetails=Impossible de charger les informations de version détaillées.
 update.version=Version
 
 # Update priority levels
-update.priority.urgent=URGENT
-update.priority.normal=NORMAL
-update.priority.minor=MINOR
-update.priority.low=LOW
+update.priority.urgent=URGENTE
+update.priority.normal=NORMALE
+update.priority.minor=MINEURE
+update.priority.low=BASSE
 
 # Breaking changes text
-update.breakingChanges=Breaking Changes:
-update.breakingChangesDefault=This version contains breaking changes
-update.migrationGuide=Migration Guide
+update.breakingChanges=Changements critiques:
+update.breakingChangesDefault=Cette version contient des changements critiques
+update.migrationGuide=Guide de migration
+
```

Comment:
The review was very meticulous and therefore identified the line as an "error," but it's not a problem. It will be corrected later in the audit.